### PR TITLE
RN-632: Show Data-Tables in the Admin-Panel

### PR DESCRIPTION
**Pull request by rohan-bes**
### Issue RN-632:

Just display data-tables for now, as we don't want users editing/deleting the internal ones. Next up is to add an interface to view the data-table data in the Admin-Panel in RN-647

### CODE REVIEW

File: packages/admin-panel/src/routes.js [hunk 1]
```diff
@@ -121,6 +122,11 @@ export const ROUTES = [
         to: '/indicators',
         component: IndicatorsPage,
       },
+      {
+        label: 'Data-Tables',
```
Comment (rohan-bes) [inline packages/admin-panel/src/routes.js:126]: Do we like this naming style? With the `-`? I kinda like it as it makes clear that `Data-Tables` is a single concept, but it's not particularly convention with Tupaia to use `-`...

File: packages/central-server/src/apiV2/dataTables/assertDataTablePermissions.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { hasBESAdminAccess } from '../../permissions';
+
+const getPermissionListWithWildcard = async accessPolicy => {
+  const userPermissionGroups = accessPolicy.getPermissionGroups();
+  return ['*', ...userPermissionGroups];
+};
```
Comment (rohan-bes) [inline packages/central-server/src/apiV2/dataTables/assertDataTablePermissions.js:11]: This code isn't the DRY-est... it's copy pasted from dataElements permissions stuff. It's a bit awkward to consolidate, but I could make more of an effort if the reviewer feels it's worth it

File: packages/admin-panel/src/routes.js (hunk 1 above)
Comment (IgorNadj) [reply packages/admin-panel/src/routes.js:126]: I don't get it sorry 😆 
But it's just a label, - or no -, it's a + to me

File: packages/central-server/src/apiV2/dataTables/assertDataTablePermissions.js (hunk 2 above)
Comment (IgorNadj) [reply packages/central-server/src/apiV2/dataTables/assertDataTablePermissions.js:11]: Sure no DRY is fine here

**Review by IgorNadj — APPROVED** [review]

**Pull request merged**